### ISSUE TITLE
Export plotly data and layout as json

### ISSUE THREAD

**Issue by Peter9192** [haddocking/haddock3#631]
#611 adds a nice webpage with plotly graphs and tables. To embed these in another page (https://github.com/i-VRESSE/bartended-haddock3) we could use https://plotly.com/javascript/react/. For that, it would be helpful to not just output the reports as html, but also as json.

Perhaps the nicest option would be that the report.html also uses report.json to get the figure layout and data.

It might be necessary to have a separate json file for each plotly object.

**Comment by mgiulini** [comment]
hi @Peter9192 , exporting the report.html in a .json file should be easy. Is it better to have a separate .json file for each .html for your purposes?
Since the analysis will be used also by command-line users I would create a `json` folder in each analysis subfolders, so that the main analysis folder doesn't get more and more crowded.

**Comment by mgiulini** [comment]
hi @Peter9192 , is this still relevant?

**Comment by Peter9192** [comment]
Not sure what direction @SarahAlidoost and @sverhoeven are taking now

**Comment by SarahAlidoost** [comment]
@mgiulini yes, this is still relevant to improve the `report.html` generated by `analyse` command. We might export plots to `json` as well.
